client/packages/invoices/src/Prescriptions/DetailView/SidePanel/SidePanel.tsx

```diff
@@ -55,16 +60,30 @@ export const SidePanelComponent = () => {
     },
   });
 
+  const onCancel = useConfirmationModal({
+    onConfirm: () => update({ status: InvoiceNodeStatus.Cancelled }),
+    title: t('heading.are-you-sure'),
+    message: t('messages.confirm-cancel-prescription'),
+  });
+
   return (
     <DetailPanelPortal
       Actions={
         <>
-          <DetailPanelAction
-            icon={<DeleteIcon />}
-            title={t('label.delete')}
-            onClick={onDelete}
-            disabled={!canDelete}
-          />
```

Comment:
Since we show *either* "Delete" or "Cancel", makes sense for Delete to conditionally appear rather than be disabled sometimes.